### DISCUSSION TITLE
New OTP component

### DISCUSSION THREAD

**Discussion by TKDev7** [mantinedev/mantine#391] [Feedback]
Like this:

![image](https://user-images.githubusercontent.com/30705623/140619100-b64c3830-3704-4514-9166-ee38a2448c0a.gif)

**Comment by rtivital** [comment]
Yes, we will add such a component in one of the future releases

**Reply by clicktodev** [reply]
Is there any movement on this?

**Reply by rtivital** [reply]
It will be added in one of the 5.x releases or in 6.0
